.buildkite/pipelines/flaky_tests/groups.json

```diff
@@ -87,6 +87,18 @@
     {
       "key": "cypress/apm_cypress",
       "name": "APM - Cypress"
+    },
+    {
+      "key": "cypress/inventory_cypress",
```

Comment:
These are not the tests you added, correct? Do they appear here because the change from other PR was not backported to `8.x`?

Reply:
discussed in DM, this should be removed as inventory_tests were not backported to 8.16